src/components/Status/StatusPill.tsx

```diff
@@ -53,7 +53,11 @@ export default function StatusPill({ className, status, size, isLink, color }: P
   )
 
   if (isLink) {
-    return <Link href={href}>{component}</Link>
+    return (
+      <Link href={href} className="Pill__Link">
```

Comment:
Can't we refactor Pill so it can receive an href instead?

Reply:
Created issue #1306